### ISSUE TITLE
Grouping in tabs in one space

### ISSUE THREAD

**Issue by Anandajoti** [tjackenpacken/taskbar-groups#131]
This is a suggestion for this great program. Let me first describe the problem (and I found this program by searching for something that may be able to do it). At present I use Messenger, WhatsApp, Twitter and Telegram, all in one "messaging" browser (see attachment). 
All four have native apps for windows whose functionality is better then the websites. The reason I do it this way is because I have only one icon open, and one space, even though there are four tabs. 
The way this program works at present is that I can group the icons, but they open in their own separate spaces and produce four new icons. 
What I would like to see from this program is a way to group arbitrary icons (as it does now), and open them all in tabs in a **single new space** of its own, like a browser does. That way we cut down on the use of real estate on the window, and also have the programs grouped in their own space. 
Would this be interesting to work on? 

![2021-06-04 06 54 56](https://user-images.githubusercontent.com/6820865/120728185-8f532a80-c506-11eb-8d16-acea90de209d.jpg)

**Comment by PikeNote** [comment]
To clarify, just so I have grasped everything and make sure this is your idea.

Like Twitter, Telegram, WhatsApp, and Messenger, when you open it from the group, you want it to open in like a single window of its own with the 4 programs grouped together. This way, you can switch between the 4 programs like tabs on a browser. While I do see what you mean, I have to explore if that is currently possible and get back to you. It does seem a bit out of the scope but is quite interesting to work on.

**Comment by Anandajoti** [comment]
Yeah, that is what I envisage. But like you don't know how possible that is. But it would be pretty revolutionary if it could be done. Hope it works out! Also though if it is not possible please just leave a note so we know.

**Comment by Anandajoti** [comment]
While still trying to solve this problem I cam across this, which shows that it is possible to group programs in tabs. https://www.nurgo-software.com/products/tidytabs

**Comment by PikeNote** [comment]
Will work on soon, I think I have discovered a more feasible solution for such case and will go ahead implement it as an optional thing. You can track when that is done [here](https://github.com/PikeNote/taskbar-groups-pike-beta/issues/8) or you can do it here.

**Comment by jterrain** [comment]
This application might be of interest:
https://www.stardock.com/products/groupy
